paragon/_variables.scss

```diff
@@ -1082,9 +1082,9 @@ $toast-box-shadow:                  $level-4-box-shadow !default;
 
 $badge-font-size:                   0.75rem !default;
 $badge-font-weight:                 400 !default;
-$badge-padding-y:                   .25rem !default;
-$badge-padding-x:                   .75rem !default;
-$badge-border-radius:               10rem !default;
```

Comment:
Let's remove this here and make in the default in the Paragon core. https://github.com/edx/paragon/blob/master/src/Badge/_variables.scss

Reply:
@abutterworth done